lib/routers/user.js

```diff
@@ -71,11 +72,19 @@ module.exports = (settings) => {
   });
 
   router.get('/', (req, res, next) => {
-    if (!req.user.profile.emailConfirmed) {
-      res.response = pick(req.user.profile, 'id', 'firstName', 'lastName', 'email');
-      return next('router');
-    }
-    next();
+    const params = {
+      model: 'profile',
+      action: 'updateLastLogin',
+      id: req.user.profile.id
+    };
+    req.workflow.update(params)
+      .then(() => {
+        if (!req.user.profile.emailConfirmed) {
+          res.response = pick(req.user.profile, 'id', 'firstName', 'lastName', 'email');
+          return next('router');
+        }
+        next()
+      });
```

Comment:
We probably want to do the "last login" bit only if the email address is confirmed.